Cargo.toml

```
@@ -1,9 +1,65 @@
-[workspace]
-
-members = ["common", "acvm_backend_barretenberg", "aztec_backend_wasm"]
-
-[workspace.package]
+[package]
+name = "acvm-backend-barretenberg"
+version = "0.1.0"
```

```suggestion
version = "0.0.0"
```
So we can make the first release with RP and it'll be 0.1.0